src/translations/messages.pot

```diff
@@ -3196,7 +3196,7 @@ msgstr ""
 msgid "Each time you open a position on an index symbol, you can start with a minimum volume transaction as indicated in the table above."
 msgstr ""
 
-msgid "Each transfer is subject to a %1 transfer fee"
+msgid "Each transfer is subject to a %1 transfer fee, minimum 0.01 charges"
```

Comment:
you don't need to update this file manually. this will automatically get updated once we run the translation update script.